php/libraries/User.class.inc

```diff
@@ -362,6 +362,45 @@ class User extends UserPermissions
         return $user_study_sites;
     }
 
+    /**
```

Comment:
the function `getStudySites()` above should be deprecated, and should be made to use this function instead with the `$study_site`

Reply:
what you should do here is probably replace the content of the function with simply a call to `getSiteList(true,true);` and add an error_log warning stating that this function is deprecated and will be removed in a near future